optimize/client/src/style.scss

```diff
@@ -1,14 +1,14 @@
-@use '@carbon/react' with (
-  $css--reset: false,
```

Comment:
Why do we need to remove the CSS reset?
This needs to be handeld carefully as it current removing the padding in a lot of element in the application. I can see this just by looking at some of the filters. Probably also the <b> tags will not work anymore.

Let's handle this in a seperate issue if possible.

Reply:
I didn't intend to remove it, I was just checking how to make the carbon font work so we can remove the fort file from `public` and I forgot to bring it back